distribution/scenes/textures/patterns/crackle2.pov

```diff
@@ -2,207 +2,189 @@
 camera {
-   location    <7, 24, 12>
-   direction   y
-   sky         z
-   up          z
-   right       2.5*x
-   look_at     <0.0, 0, 0.3>
-   angle       30
+    location  <7,24,12>
+    direction y
+    sky       z
+    up        z
+    right     2.5*x
+    look_at   <0,0,0.3>
+    angle     30
 }
 
-
+#declare SkyColor = srgb <0.60,0.70,0.95>;
 sphere {
-   <0, 0, 0>, 1
-   texture {
-      pigment {
-         color rgb < 0.60, 0.70, 0.95 >
-      }
-      finish {
-         diffuse 0
-         ambient 1
-      }
-   }
-   scale 100000
-   no_shadow
-   hollow on
+    <0,0,0>, 1
+    texture {
+        pigment { SkyColor }
+        finish {
+            ambient  0
+            diffuse  0
+            emission 1
+        }
+    }
+    scale 100000
+    no_shadow
+    hollow on
 }
 
+#declare FloorColorA = srgb <0.63137,0.60392,0.56078>;
+#declare FloorColorB = srgb <0.80392,0.73725,0.73725>;
+#declare FloorColorC = srgb <0.87843,0.81176,0.81176>;
+#declare FloorColorD = srgb <0.92549,0.89804,0.87843>;
 plane {
-   z, 0
-   
-   texture {
-      pigment {
-         crackle
-         color_map {
-            [ 0.0 color rgb<0.356, 0.321, 0.274> ]
-            [ 0.1 color rgb<0.611, 0.500, 0.500> ]
-            [ 0.4 color rgb<0.745, 0.623, 0.623> ]
-            [ 1.0 color rgb<0.837, 0.782, 0.745> ]
-         }
-         warp { turbulence 0.6 }
-      }
-      finish {
-         diffuse 0.6
-         ambient 0.1
-         specular 0.2
-         reflection {
-            0.2, 0.6
-            fresnel on
-         }
-         conserve_energy
-      }
-    
+    z, 0
+    texture {
+        pigment {
+            crackle
+            color_map {
+                [ 0.0 FloorColorA ]
+                [ 0.1 FloorColorB ]
+                [ 0.4 FloorColorC ]
+                [ 1.0 FloorColorD ]
+            }
+            warp { turbulence 0.6 }
+        }
+        finish {
+            diffuse 0.6
+            ambient 0.006
+            specular 0.2
+            reflection {
+                0.2, 0.6
+                fresnel on
+            }
+            conserve_energy
+        }
    }
-   
 }
 
 #macro Objs(Metr)
-
-union {
-   #local fn_Crack=
-   function {
-      pigment {
-         crackle
-         metric Metr
-         color_map { [0 rgb 0][1 rgb 1] }
-         scale 0.7
+  union {
+      #local T_CrackColor00 = srgb <0.52157,0.40000,0.72549>;
+      #local T_CrackColor01 = srgb <0.83137,0.66667,0.92157>;
+      #local T_CrackColor02 = srgb <0.83137,0.66667,0.92157>;
+      #local T_CrackColor03 = srgb <0.87059,0.66667,0.92941>;
+      #local T_CrackColor04 = srgb <0.95686,0.75294,0.95294>;
+      #local T_CrackColor05 = srgb <1.00000,0.84314,0.93333>;
+      #local T_CrackColor06 = srgb <0.98823,0.85490,0.79216>;
+      #local T_CrackColor07 = srgb <0.97647,0.86667,0.67059>;
+      #local T_CrackColor08 = srgb <0.98823,0.86667,0.60784>;
+      #local T_CrackColor09 = srgb <0.58431,0.48628,0.90588>;
```

Comment:
Somewhat agree about the disconnect here, but there are reasons. 

1) This is the form I use and others on the newsgroups have adopted so the multiplication of the color - often done as a quick tweak - actually works as the user expects. This scene doesn't have it but others do multiplication in the color_map from defined colors like this to show the form which works.  

2) Normally the color names would mean something is this form as in SVG_redorange, but in trying to match 3.6 results as closely as possible I was stuck with either going with generic names like those above or coming up with names that might be close to some web color people might know. 

3) Not true here, but very often colors are repeated in color maps and a common definition makes this much more clear.